wallets/rn_cli_wallet/src/screens/SubscriptionDetailsScreen.tsx

```diff
@@ -0,0 +1,109 @@
+import {useEffect, useState} from 'react';
+import {useRoute} from '@react-navigation/native';
+import {Alert, FlatList, View} from 'react-native';
+import useNotifyClientContext from '@/hooks/useNotifyClientContext';
+
+import NotificationItem from '@/components/NotificationItem';
+import NotificationItemSkeleton from '@/components/NotificationItemSkeleton';
+
+interface NotifyNotification {
+  title: string;
+  sentAt: number;
+  body: string;
+  id: string;
+  url: string | null;
+  type: string;
+}
+
+export default function SubscriptionDetailsScreen() {
+  const {params} = useRoute();
+  const {notifications, setNotifications} = useNotifyClientContext();
+  const [hasMore, setHasMore] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const topic = params?.topic;
+
+  const topicNotifications = notifications[params?.topic];
+  const sortedByDate = topicNotifications?.sort(
+    (a: NotifyNotification, b: NotifyNotification) => b.sentAt - a.sentAt,
+  );
+  const lastItem = sortedByDate?.[sortedByDate.length - 1]?.id;
+
+  const {account, notifyClient} = useNotifyClientContext();
+
+  async function getNotificationHistory(startingAfter?: string) {
```

Comment:
We could separate these business logics into hooks but we already have `@web3inbox/hooks` package and we are planning to have a refactor if it and we could start using those hooks, so we don't have to duplicate work rn. Lets keep them